lib/iam_ssoe_oauth/session_manager.rb

```diff
@@ -53,6 +53,7 @@ def create_user_session
       user = build_user(user_identity)
       handle_nil_user(user_identity) if user.nil?
       validate_user(user)
+      create_evss_account(user) if user.authorize(:evss, :access?)
```

Comment:
I couldn't add the code in our ApplicationController#authorize because it just returns the user after session creation and you can't tell if it's a newly created user session or an existing session that was pulled from memory. We only want this to happen on login, not on authorization of every single request. 

The other option was to add it to the users controller show method, but that is probably called more often than just on login as well.